facia/app/controllers/FaciaController.scala

```diff
@@ -171,6 +171,26 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
       }
   }
 
+  def renderShowMore(path: String, collectionId: String) = MemcachedAction { implicit request =>
+    for {
+      maybeFaciaPage <- frontJson.get(path)
+    } yield {
+      val maybeResponse = for {
+        faciaPage <- maybeFaciaPage
+        (container, index) <- faciaPage.front.containers.zipWithIndex.find(_._1.dataId == collectionId)
```

Comment:
We have to because of de-duplication, etc. anyway. As far as perf bottlenecks go I doubt it's an issue, though. Our servers are extremely beefy, and the difference between partially transforming a front into objects or only transforming a section of it is not going to be huge.